tslearn/clustering.py

```diff
@@ -915,8 +918,12 @@ class KShape(BaseEstimator, ClusterMixin,
     .. [1] J. Paparrizos & L. Gravano. k-Shape: Efficient and Accurate
        Clustering of Time Series. SIGMOD 2015. pp. 1855-1870.
     """
+
+    model_attrs = ['inertia_', 'cluster_centers_', '_X_fit', '_norms', '_norms_centroids']
```

Comment:
Define a class attribute which is the list of model attributes that are sufficient to describe the model. BaseModelPackage will throw an exception if it doesn't find this class attribute.